src/cli/README_SRP.md

```diff
@@ -0,0 +1,181 @@
+# OpenThread CLI - SRP (Service Registration Protocol)
+
+## Quick Start
+
+### Build with SRP Server & Client support
+
+Use the `SRP_SERVER=1 SERVICE=1 ECDSA=1` build switches to enable SRP Server API support, and `SRP_CLIENT=1 ECDSA=1` build switches for SRP Client API support.
+
+Make simulation SRP Server & Client:
+
+```bash
+./bootstrap
+make -f examples/Makefile-simulation SRP_SERVER=1 SERVICE=1 SRP_CLIENT=1 ECDSA=1
+```
+
+## Simulation Example
+
+### Start SRP Server
+
+start the SRP Server node with below command:
+
+```bash
+./output/simulation/bin/ot-cli-ftd 1
+```
+
+Execute below command to setup a Thread network and start the SRP Server:
```

Comment:
For most of these, you can leave out "below command" references... just state the action.

"Set up a Thread network and start the SRP Server:"